strings-csharp/CountNumberOfVowelsInString/CountNumberOfVowelsInString/VowelCounters.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System.Text.RegularExpressions;
+
+public class VowelCounters
+{
+    static void Main()
+    {
+        var vowels = new HashSet<char> { 'A', 'a', 'E', 'e', 'I', 'i', 'O', 'o', 'U', 'u' };
+        string sentence = "In the vast expanse of the universe, countless galaxies swirl in a cosmic dance, each telling a unique story of creation and destruction.";
+        Console.WriteLine(sentence);
+
+        CountVowelsWithForLoop(sentence, vowels);
+        CountVowelsWithForEachLoop(sentence, vowels);
+        CountVowelsUsingSwitchStatement(sentence);
+        CountVowelsUsingRegEx(sentence);
+        CountVowelsUsingStrReplaceAndLength(sentence);
+        CountVowelsUsingLinq(sentence, vowels);
+    }
+
+    public static int CountVowelsWithForLoop(string sentence, HashSet<char> vowels)
+    {
+        int total = 0;
```

Comment:
Use var instead of explicit datatype "int"
Same comments throughout the solution